js/battle.js

```diff
@@ -4004,43 +4008,50 @@ var Battle = (function () {
 					}
 					break;
 				default:
+					this.resultAnim(poke, "Acquired " + ability.name, 'ability', animDelay);
 					actions += "" + poke.getName() + " acquired " + ability.name + "!";
 					break;
-				} else switch (ability.id) {
-				case 'pressure':
-					actions += "" + poke.getName() + " is exerting its pressure!";
-					break;
-				case 'moldbreaker':
-					actions += "" + poke.getName() + " breaks the mold!";
-					break;
-				case 'turboblaze':
-					actions += "" + poke.getName() + " is radiating a blazing aura!";
-					break;
-				case 'teravolt':
-					actions += "" + poke.getName() + " is radiating a bursting aura!";
-					break;
-				case 'intimidate':
-					actions += '' + poke.getName() + ' intimidates ' + ofpoke.getLowerName() + '!';
-					break;
-				case 'unnerve':
-					actions += "" + poke.getName() + "'s Unnerve makes " + this.getSide(args[3]).getLowerTeamName() + " too nervous to eat Berries!";
-					break;
-				case 'aurabreak':
-					actions += "" + poke.getName() + " reversed all other Pokémon's auras!";
-					break;
-				case 'fairyaura':
-					actions += "" + poke.getName() + " is radiating a fairy aura!";
-					break;
-				case 'darkaura':
-					actions += "" + poke.getName() + " is radiating a dark aura!";
-					break;
-				case 'airlock':
-				case 'cloudnine':
-					actions += "The effects of the weather disappeared.";
-					break;
-				default:
-					actions += "" + poke.getName() + " has " + ability.name + "!";
-					break;
+				} else {
+					actions += "[" + poke.getName() "'s " + ability.name + "]<br>";
+					this.resultAnim(poke, ability.name, 'ability', animDelay);
+					switch (ability.id) {
+					case 'pressure':
+						actions += "" + poke.getName() + " is exerting its pressure!";
+						break;
+					case 'moldbreaker':
+						actions += "" + poke.getName() + " breaks the mold!";
+						break;
+					case 'turboblaze':
+						actions += "" + poke.getName() + " is radiating a blazing aura!";
+						break;
+					case 'teravolt':
+						actions += "" + poke.getName() + " is radiating a bursting aura!";
+						break;
+					case 'intimidate':
+						actions += '' + poke.getName() + ' intimidates ' + ofpoke.getLowerName() + '!';
+						break;
+					case 'unnerve':
+						actions += "" + poke.getName() + "'s Unnerve makes " + this.getSide(args[3]).getLowerTeamName() + " too nervous to eat Berries!";
+						break;
+					case 'aurabreak':
+						actions += "" + poke.getName() + " reversed all other Pokémon's auras!";
+						break;
+					case 'fairyaura':
+						actions += "" + poke.getName() + " is radiating a fairy aura!";
+						break;
+					case 'darkaura':
+						actions += "" + poke.getName() + " is radiating a dark aura!";
+						break;
+					case 'airlock':
+					case 'cloudnine':
+						actions += "The effects of the weather disappeared.";
+						break;
+					default:
+						if (!kwargs.activate) {
```

Comment:
AFAIK, `POKEMON has ABILITY` is used in exactly one case: Gen 4 Mold Breaker (which means nothing in XY). @Marty-D should verify, though.